lib/src/virtual_host.dart

```diff
@@ -44,23 +44,23 @@ abstract class VirtualHost {
 }
 
 class _VirtualHostDomain {
-  StreamController<HttpRequest> any;
-  StreamController<HttpRequest> exact;
+  StreamController<HttpRequest>? any;
+  StreamController<HttpRequest>? exact;
   Map<String, _VirtualHostDomain> subDomains = {};
 }
 
 class _VirtualHost implements VirtualHost {
   final _VirtualHostDomain _topDomain = _VirtualHostDomain();
-  StreamController<HttpRequest> _unhandledController;
+  StreamController<HttpRequest>? _unhandledController;
```

Comment:
```suggestion
  late final _unhandledController =  StreamController<HttpRequest>();
```

Late is also lazy so it has the same behavior as we had before :)

Reply:
See the comments below. It might prevent it from handling both an un-handled and handled request.

Reply:
Ya I didn't see that code that was relying on the ability to null check, so you can leave this as is :)